sklearn-demo.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
<p>small data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/maxjakob/reviewnb-playground/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>if only someone would write a tool that recognizes or <strong>ignores</strong> such <em>cross-cell</em>  changes.</p><p> </p><blockquote>sdfdsf</blockquote><p> </p><p><code>sdfdsfd</code></p><p> </p><p><code><a href="https://www.ecosia.org" target="_blank">sdfdsf</a></code></p><p> </p><h1>dfsdf</h1><h2>sfdsdf</h2><h3>sdfsd</h3>